pkg/util/errors.go

```diff
@@ -324,44 +323,38 @@ func HandleInstanceSharingError(err error, entityName string) error {
 	case ErrChangingParametersOfReferenceInstance:
 		return &HTTPError{
 			ErrorType:   "BadRequest",
-			Description: fmt.Sprintf("Failed to update the instance %s. It isn't allowed to change a reference instance.", entityName),
+			Description: fmt.Sprintf("Failed to update the instance %s. It is not allowed to change the parameters of a reference instance.", entityName),
 			StatusCode:  http.StatusBadRequest,
 		}
 	case ErrMissingOrInvalidReferenceParameter:
 		return &HTTPError{
 			ErrorType:   "BadRequest",
-			Description: fmt.Sprintf("Failed to create the instance. Missing or invalid parameter \"%s\".", entityName),
-			StatusCode:  http.StatusBadRequest,
-		}
-	case ErrUnknownOSBMethod:
-		return &HTTPError{
-			ErrorType:   "BadRequest",
-			Description: fmt.Sprintf("Unknown OSB method: \"%s\".", entityName),
+			Description: "Failed to create the instance. Specify an instance ID or use search criteria.", // consider to mention the help.sap documentation on the plan's description.
```

Comment:
Just this one message still seems strange to me.
Why did it fail?
"Specify the instance ID or use search criteria?"
Something here doesn't seem right.